packages/shared/src/components/filters/FilterRedDot.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { ReactElement, useContext } from 'react';
 import SettingsIcon from '../../../icons/settings.svg';
-import RedDot from '../RedDot';
+import { RedDot } from '../notifs';
```

Comment:
Please change `notifs` to something more explicit

Reply:
And change RedDot to AlertDot and accept the color attribute

Reply:
Let me change this to `AlertDot` for now. The color attribute can be tricky at some point in the future, let's consult @tsahimatsliah on that part.